Makefile

```diff
@@ -185,7 +185,7 @@ clean-cross-build:
 build: clean_build_local _build_local copy_build_local
 .PHONY: build
 
-_build_local: tidy-vendor format
+_build_local:
```

Comment:
I think CI must check if vendor is up to date and that we don't accept patches that have difference in `vendor` directory if it isn't up to date.  With vendoring there is a potential for someone to accidentally add code they may have used for testing by editing the vendor/ files.